src/canmatrix/canmatrix.py

```diff
@@ -175,7 +175,7 @@ def multiplexSetter(self, value):
             ret_multiplex = int(value)
             self.mux_val = int(value)
         else:  # is it valid for None too?
-            self.is_multiplexer = True
+            self.is_multiplexer = False
```

Comment:
I'm not clear why this should change.  Can you explain?

Reply:
I did some more investigation and provide a better fix in PR #213.